src/serac/numerics/tests/CMakeLists.txt

```diff
@@ -17,7 +17,11 @@ serac_add_tests( SOURCES ${numerics_serial_tests}
                  NUM_MPI_TASKS 1)
 
 if(PETSC_FOUND)
-    serac_add_tests(SOURCES       equationsolver_petsc.cpp
+    set(petsc_solver_tests
```

Comment:
if this test requires slepc, it may make more sense to check `if(SLEPC_FOUND)` nested inside the petsc check.

Reply:
I adjusted the cmake logic.  please double check it makes sense!  thanks